### ISSUE TITLE
Round theme breaking Theme Manager

### ISSUE THREAD

**Issue by Titaniumtown** [EMERALD0874/Steam-Deck-Themes#11]
Installing the round theme breaks the theme store so many elements do not appear:
![image](https://user-images.githubusercontent.com/11786225/181756435-337d2217-460d-467a-a310-d01e77d7cf46.png)
![image](https://user-images.githubusercontent.com/11786225/181756457-6696ac57-059b-4ab0-b7ec-b8f69c03313a.png)

Can reproduce on both Beta and Stable.

**Comment by Ian-Boraks** [comment]
Problem was that:
```css
.gamepaddialog_GamepadDialogContent_3joNk {
  border-radius: 20px;
}
```
Was modifying the mod page, as the mod uses the dialog content to create the window which the themes are displayed in.
Fix:
```css
.gamepaddialog_GamepadDialogContent_3joNk:not(.gamepadpagedsettings_PagedSettingsDialog_PageContent_wIOLO) {
  border-radius: 20px;
}
```

**Comment by EMERALD0874** [comment]
Closed by #12
